io.catenax.special_characteristics.measurement/1.0.0/Measurement.ttl

```diff
@@ -0,0 +1,83 @@
+
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.special_characteristics.measurement:1.0.0#> .
+@prefix bpn: <urn:samm:io.catenax.shared.business_partner_number:2.0.0#> .
+@prefix samm-u: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix serialPart: <urn:samm:io.catenax.serial_part:3.0.0#> .
+
+:SpecialCharacteristicMeasurement a samm:Aspect ;
+   samm:preferredName "SpecialCharacteristicMeasurment"@en ;
+   samm:description "Contains the Measurment for a special characteristic"@en ;
+   samm:properties ( serialPart:localIdentifiers bpn:bpnlProperty serialPart:customerPartId :characteristicId :results [ samm:property :measurementType; samm:optional true ] :revisionIndex ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:characteristicId a samm:Property ;
+   samm:preferredName "characteristicId"@en ;
+   samm:description "Identification of the special characteristic in in relation of the specified part"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "S1" .
+
+:results a samm:Property ;
+   samm:preferredName "measurement_results"@en ;
+   samm:description "List of measurementObjects"@en ;
+   samm:characteristic :ResultList .
+
+:measurementType a samm:Property ;
+   samm:preferredName "Measurement Type"@en ;
+   samm:description "This optional attribute hints the type of the measurement"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "simpleMeasurement" .
+
+:revisionIndex a samm:Property ;
+   samm:preferredName "Revision Index"@en ;
+   samm:description "This attribute contains the revision index of the special characteristic, it is necessary to identify a special characteristic"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "01" .
+
+:ResultList a samm-c:List ;
+   samm:preferredName "ResultList"@en ;
+   samm:description "List of result objects"@en ;
+   samm:dataType :MeasurmentObject .
+
+:MeasurmentObject a samm:Entity ;
+   samm:preferredName "Measurmentobject"@en ;
+   samm:description "This Entity represents a generic Measurment Object"@en ;
+   samm:properties ( :identifier :timestamp :value :unit :description ) .
+
+:identifier a samm:Property ;
+   samm:preferredName "Identifier"@en ;
+   samm:description "Use this Attribute to identify the measuremnt "@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "measurementPoint23" .
+
+:timestamp a samm:Property ;
```

Comment:
Don't use a generic name for this property. If you want to describe a measurement time, name it that way.

Reply:
done